src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -79,34 +87,38 @@ function DomainSlider(props: Props): ReactElement {
       />
 
       <ToggleBtn
-        small
-        label={getAutoscaleLabel(isAutoMin, isAutoMax)}
-        icon={FiZap}
-        value={isAutoscaling}
+        iconOnly
+        label="Edit domain"
+        icon={MdTune}
+        value={isEditing}
         disabled={disabled}
-        onChange={() => {
-          onCustomDomainChange(
-            isAutoscaling ? dataDomain : [undefined, undefined]
-          );
-        }}
+        onChange={() => toggleAll(!isEditing)}
       />
 
       <DomainTooltip
         id={TOOLTIP_ID}
-        open={tooltipOpen}
-        domain={sliderDomain}
+        open={hovered || isEditing}
+        sliderDomain={sliderDomain}
         dataDomain={dataDomain}
         errors={errors}
         isAutoMin={isAutoMin}
         isAutoMax={isAutoMax}
         onAutoMinToggle={() => {
           const newMin = isAutoMin ? dataDomain[0] : undefined;
           onCustomDomainChange([newMin, customDomain[1]]);
+          toggleEditingMin(isAutoMin);
```

Comment:
Not sure about this. It feels odd to toggle editing when deactivating the autoscale.

My expectation would be that the focus on the edition is lost when toggling a button (i.e. `toggleEditingMin(false)`).

Reply:
Oh yeah? I thought maybe users who disable the autoscale would want to edit the value right away in some cases. That being said, it does "pin" the popup, which means an extra click is required to close it if they're not interested in editing the value... Alright, I'll change this.

Reply:
> That being said, it does "pin" the popup, which means an extra click is required to close it if they're not interested in editing the value... Alright, I'll change this.

Yup that was my point

Reply:
However, I think if a field is in edit mode and the user disables the autoscale, I think the field should remain in edit mode.